build.sh

```diff
@@ -1,5 +1,5 @@
 mkdir build
 cd build
-cmake .. -DCOMPUTECPP_PACKAGE_ROOT_DIR=$1 -DCMAKE_BUILD_TYPE=Release -DOPENBLAS_ROOT=/tmp/OpenBLAS/build
+cmake .. -DCOMPUTECPP_PACKAGE_ROOT_DIR=$1 -DCMAKE_BUILD_TYPE=Release -DBLAS_DIR=/tmp/OpenBLAS/build
```

Comment:
if you use this script for travis, you shouldn't need the hack inside.
If you need it, then please add a comment explaining why the hack is there